Serial doesn't work for mbed framework

Example
```c
#include "mbed.h"

Serial pc(USBTX, USBRX); // tx, rx

int main() {
    pc.printf("Hello World!\n\r");
    while(1) {
        pc.putc(pc.getc() + 1); // echo input back to terminal
    }
}
```

```shell
(develop)➜  mbed-blink git:(develop) ✗ platformio run -e nucleo_f401re -t upload
[Fri Mar 13 23:04:44 2015] Processing nucleo_f401re (platform: ststm32, board: nucleo_f401re, framework: mbed)
----------------------------------------------------------------------------------------------------------------------------------------------------------------
arm-none-eabi-g++ -o .pioenvs/nucleo_f401re/src/main.o -c -std=gnu++98 -fno-rtti -fdata-sections -ffunction-sections -Wno-unused-parameter -fno-exceptions -Wextra -fno-delete-null-pointer-checks -fmessage-length=0 -mthumb -Wno-missing-field-initializers -c -fno-builtin -O2 -mfpu=fpv4-sp-d16 -fomit-frame-pointer -Wall -mfloat-abi=softfp -MMD -mcpu=cortex-m4 -DTARGET_NUCLEO_F401RE -DTARGET_M4 -DTARGET_CORTEX_M -DTARGET_STM -DTARGET_STM32F4 -DTARGET_STM32F401RE -DTOOLCHAIN_GCC_ARM -DTOOLCHAIN_GCC -D__CORTEX_M4 -DARM_MATH_CM4 -D__FPU_PRESENT=1 -DMBED_BUILD_TIMESTAMP=1426262453.27 -D__MBED__=1 -DTARGET_FF_ARDUINO -DTARGET_FF_MORPHO -DPLATFORMIO=010200 -I.pioenvs/nucleo_f401re/FrameworkMbedInc248832578 -I.pioenvs/nucleo_f401re/FrameworkMbedInc-213564679 -I.pioenvs/nucleo_f401re/FrameworkMbedInc-250686161 -I.pioenvs/nucleo_f401re/FrameworkMbedInc1266984661 -I.pioenvs/nucleo_f401re/FrameworkMbedInc2097404638 -I.pioenvs/nucleo_f401re/FrameworkMbedInc-1116590620 -I.pioenvs/nucleo_f401re/FrameworkMbedInc170856846 -I.pioenvs/nucleo_f401re/FrameworkMbedInc175333816 .pioenvs/nucleo_f401re/src/main.cpp
arm-none-eabi-ar rcs .pioenvs/nucleo_f401re/libFrameworkMbed.a @/var/folders/nj/nt38kxwd2xxfl5l42np_w7qw0000gn/T/pioarargs-f8d149b23e80c5a2dfe687bd465da31f
arm-none-eabi-ranlib .pioenvs/nucleo_f401re/libFrameworkMbed.a
arm-none-eabi-g++ -o .pioenvs/nucleo_f401re/firmware.elf -Wl,--gc-sections -Wl,--wrap,main -mcpu=cortex-m4 -mthumb -mfpu=fpv4-sp-d16 -mfloat-abi=softfp --specs=nano.specs -T /Users/ikravets/.platformio/packages/framework-mbed/variant/NUCLEO_F401RE/mbed/TARGET_NUCLEO_F401RE/TOOLCHAIN_GCC_ARM/STM32F401XE.ld .pioenvs/nucleo_f401re/src/main.o -L/Users/ikravets/.platformio/packages/framework-mbed/variant/NUCLEO_F401RE/mbed/TARGET_NUCLEO_F401RE/TOOLCHAIN_GCC_ARM -L.pioenvs/nucleo_f401re -Wl,--start-group -lc -lgcc -lm -lmbed -lstdc++ -lsupc++ -lnosys .pioenvs/nucleo_f401re/libFrameworkMbed.a -Wl,--end-group
.pioenvs/nucleo_f401re/libFrameworkMbed.a(retarget.o): In function `__gnu_cxx::__verbose_terminate_handler()':
retarget.cpp:(.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x0): multiple definition of `__gnu_cxx::__verbose_terminate_handler()'
/Users/ikravets/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/lib/armv7e-m/softfp/libstdc++_s.a(vterminate.o):vterminate.cc:(.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x0): first defined here
/Users/ikravets/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/lib/armv7e-m/softfp/libc_s.a(lib_a-signalr.o): In function `_kill_r':
signalr.c:(.text._kill_r+0xe): undefined reference to `_kill'
/Users/ikravets/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/lib/armv7e-m/softfp/libc_s.a(lib_a-signalr.o): In function `_getpid_r':
signalr.c:(.text._getpid_r+0x0): undefined reference to `_getpid'
collect2: error: ld returned 1 exit status
scons: *** [.pioenvs/nucleo_f401re/firmware.elf] Error 1
================================================================= [ ERROR ] Took 1.35 seconds =================================================================
```

--- a/examples/mbed/mbed-serial/README.rst
+++ b/examples/mbed/mbed-serial/README.rst
@@ -1,0 +1,21 @@
+How to buid PlatformIO based project
+====================================
+
+1. `Install PlatformIO <http://docs.platformio.org/en/latest/installation.html>`_
+2. Download `source code with examples <https://github.com/ivankravets/platformio/archive/develop.zip>`_
+3. Extract ZIP archive
+4. Run these commands:
+
+.. code-block:: bash
+
+    # Change directory to example
+    > cd platformio-develop/examples/mbed/mbed-serial
+
+    # Process example project
+    > platformio run
+
+    # Upload firmware
+    > platformio run --target upload
+
+    # Clean build files
+    > platformio run --target clean

--- a/examples/mbed/mbed-serial/platformio.ini
+++ b/examples/mbed/mbed-serial/platformio.ini
@@ -1,0 +1,43 @@
+#
+# Project Configuration File
+#
+# A detailed documentation with the EXAMPLES is located here:
+# http://docs.platformio.org/en/latest/projectconf.html
+#
+
+# A sign `#` at the beginning of the line indicates a comment
+# Comment lines are ignored.
+
+# Simple and base environment
+# [env:mybaseenv]
+# platform = %INSTALLED_PLATFORM_NAME_HERE%
+# framework =
+# board =
+#
+# Automatic targets - enable auto-uploading
+# targets = upload
+
+
+# NXP LPC Platform
+[env:lpc1768]
+platform = nxplpc
+framework = mbed
+board = lpc1768
+
+# Nordic nRF51 Platform
+[env:nrf51_mkit]
+platform = nordicnrf51
+framework = mbed
+board = nrf51_mkit
+
+# Freescale FRDM Platform
+[env:frdm_kl25z]
+platform = freescalekinetis
+framework = mbed
+board = frdm_kl25z
+
+# ST STM32 Platform
+[env:nucleo_f401re]
+platform = ststm32
+framework = mbed
+board = nucleo_f401re

--- a/examples/mbed/mbed-serial/src/main.cpp
+++ b/examples/mbed/mbed-serial/src/main.cpp
@@ -1,0 +1,10 @@
+#include "mbed.h"
+
+Serial pc(USBTX, USBRX); // tx, rx
+
+int main() {
+    pc.printf("Hello World!\n\r");
+    while(1) {
+        pc.putc(pc.getc() + 1); // echo input back to terminal
+    }
+}

--- a/platformio/builder/scripts/frameworks/mbed.py
+++ b/platformio/builder/scripts/frameworks/mbed.py
@@ -125,8 +125,7 @@ for lib_path in eixdata.get("CPPPATH"):
 env.Append(
     LIBPATH=[join(variant_dir, lib_path)
              for lib_path in eixdata.get("LIBPATH", [])
-             if lib_path.startswith("mbed")],
-    LIBS=["mbed"]
+             if lib_path.startswith("mbed")]
 )
 
 #
@@ -134,6 +133,7 @@ env.Append(
 #
 
 libs = [l for l in eixdata.get("STDLIBS", []) if l not in env.get("LIBS")]
+libs.append("mbed")
 
 libs.append(env.Library(
     join("$BUILD_DIR", "FrameworkMbed"),

